packages/sdk/Dockerfile

```diff
@@ -47,6 +47,11 @@ curl -sSL https://github.com/google/go-containerregistry/releases/download/v${CR
     tar -zx -C /usr/local/bin
 EOF
 
+# alto
+FROM node:18.19.0-bookworm AS alto
```

Comment:
Why using node 18?

Reply:
Because down there I'm installing `nodejs` ubuntu package, which install node 18. So I decided to use 18 in the build phase as well.